opennlp-tools/src/test/java/opennlp/tools/cmdline/tokenizer/TokenizerTrainerToolTest.java

```diff
@@ -104,7 +108,10 @@ public void testTestRunHappyCase() throws IOException {
     model.delete();
   }
 
-  @Test
+  //TODO OPENNLP-1447
+  @Disabled(value = "OPENNLP-1447: These kind of tests won't work anymore. " +
+          "We need to find a way to redirect log output (i.e. implement " +
+          "a custom log adapter and plug it in, if we want to do such tests.")
```

Comment:
:+1: that's common in Python, but I have never had to do that in Java. Maybe something like: https://stackoverflow.com/questions/29076981/how-to-intercept-slf4j-with-logback-logging-via-a-junit-test (will comment in that issue).

Reply:
The log watcher approach as outlined in the thread looks promising. I will add a reference in OPENNLP-1447

Reply:
Already documented by @kinow  -> thanks!